core/domain/story_services.py

```diff
@@ -313,6 +321,9 @@ def delete_story(committer_id, story_id, force_deletion=False):
     story_model.delete(
         committer_id, feconf.COMMIT_MESSAGE_STORY_DELETED,
         force_deletion=force_deletion)
+    for node in story.story_contents.nodes:
+        exp_services.update_exploration_story_link(
```

Comment:
Use some kind of delete_multi() operation if you are doing datastore updates in a for loop.

Reply:
Depends on previous comment. If the function is to be removed, the storage change can be manually done here with a delete_multi.

Reply:
Yes, let's do it manually here.

Reply:
Done